### ISSUE TITLE
Naming and id for a new ABNT numeric style (question)

### ISSUE THREAD

**Issue by imsys** [citation-style-language/styles#6408]
I compared all the ABNT styles by different institution, and they differ mainly on:

* Old (6023:2002) vs new (6023:2018) ABNT. The main highly noticeable difference I know is that the old has a < > around URLs, and the new doesn't. The main ABNT csl is using the old style (associacao-brasileira-de-normas-tecnicas.csl), I wonder if the year version should be added to the title to help newcomers searching for ABNT in Zotero. Both NBR 6023:2002 and 6023:2018 are about bibliography rules and both mention that the NBR 10520:2002 rules should be used for the documentation on how to do the citations. So, having the NBR 6023 version in the tile should be enough, instead of mentioning the version of both NBR 6023 and NBR 10520.

* Name abbreviation or not.
* <bibliography et-al-min and et-al-use-first, NBR 6023:2018 "et-al-min" should be 4 or higher, but recommends to show all authors.
* <citation et-al-min and et-al-use-first, there is no mentions or recommendations about those in NBR 10520-2002, but most assume et-al-min=4 and et-al-use-first=1
* disambiguate arguments or not (they all need a way to disambiguate, the ones that doesn't is probably by forgot to set it) #6364 
* some collapse the year (in citation), others don't. ABNT says it should collapse.
* Some include DOI, others don't. As stated in page 11 and 14 of NBR 6023:2018, DOI should be included whenever available. Page 11 also says ISSN should be included when available.
* Some include ABNT optional fields (ISBN, number of pages in book etc.)

- ABNT has documentation for both author-date and numeric citation format, and currently there is no public numeric 2018 ABNT style in the repository, and I would like to create and share it if possible.

**My background**: I'm an undergraduate pharmacology student at University of Marilia (UNIMAR), where the rule is to just use the standard ABNT.

**Intentions**: I noticed there is no youtube video tutorial in Portuguese on how to use Zotero with Google Docs and syncing it with other colleagues working on the same project, so I thought to create this video tutorial, and it would be really good if I could also explain in the video the differences between the current public styles available, and I would like to give them options for numeric ABNT too, but there is currently none.

Bellow is some examples of the old ABNT style, and the current best ones that best fit the 2018 ABNT:

### Associação Brasileira de Normas Técnicas (Português - Brasil) (NBR 6023:2002) (old)
`<citation et-al-min="4" et-al-use-first="1" collapse="year" disambiguate-add-year-suffix="true">`
`<bibliography hanging-indent="false" et-al-min="4" et-al-use-first="1" entry-spacing="1">`

(KWAN; BEYENE; SHAH, 2009; LOEVINSOHN, 2008; LÜCK, 2014)
KWAN, D.; BEYENE, J.; SHAH, P. S. Adverse Consequences of Internet Purchase of Pharmacologic Agents or Dietary Supplements. **Journal of Pharmacy Technology**, v. 25, n. 6, p. 355–360, nov. 2009.
LOEVINSOHN, B. **Performance-based contracting for health services in developing countries: a toolkit.** Washington, DC: World Bank, 2008.
LÜCK, H. **Liderança em gestão escolar.** São Paulo: Editora Vozes, 2014. 

### Instituto Brasileiro de Informação em Ciência e Tecnologia - ABNT (autoria abreviada) (NBR 6023:2018)
`<citation et-al-min="4" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" collapse="year-suffix" givenname-disambiguation-rule="primary-name" year-suffix-delimiter=", ">`
`<bibliography hanging-indent="false" et-al-min="21" et-al-use-last="true" et-al-use-first="19" entry-spacing="1" line-spacing="1">`

(KWAN; BEYENE; SHAH, 2009; LOEVINSOHN, 2008; LÜCK, 2014)
KWAN, D.; BEYENE, J.; SHAH, P. S. Adverse Consequences of Internet Purchase of Pharmacologic Agents or Dietary Supplements. **Journal of Pharmacy Technology**, v. 25, n. 6, p. 355–360, nov. 2009. DOI [10.1177/875512250902500602](https://doi.org/10.1177/875512250902500602). Disponível em: http://journals.sagepub.com/doi/10.1177/875512250902500602. Acesso em: 22 maio 2022.
LOEVINSOHN, B. **Performance-based contracting for health services in developing countries: a toolkit.** Washington, DC: World Bank, 2008(Health, nutrition, and population series).
LÜCK, H. **Liderança em gestão escolar.** São Paulo: Editora Vozes, 2014.

(this is different than the standard ABNT in that when the location is not available it should show [S. l.] )

### Universidade Estadual de Alagoas - UNEAL [ABNT] (Portuguese - Brazil) (NBR 6023:2018)
`<citation et-al-min="4" et-al-use-first="1" collapse="year" disambiguate-add-year-suffix="true">`
`<bibliography hanging-indent="false" et-al-min="10" et-al-use-first="1" entry-spacing="1">`

(KWAN; BEYENE; SHAH, 2009; LOEVINSOHN, 2008; LÜCK, 2014)
KWAN, Della; BEYENE, Joseph; SHAH, Prakesh S. Adverse Consequences of Internet Purchase of Pharmacologic Agents or Dietary Supplements. **Journal of Pharmacy Technology**, [S. l.], v. 25, n. 6, p. 355–360, 2009. ISSN: 8755-1225, 1549-4810. DOI: [10.1177/875512250902500602](https://doi.org/10.1177/875512250902500602). Disponível em: http://journals.sagepub.com/doi/10.1177/875512250902500602. Acesso em: 22 maio. 2022.
LOEVINSOHN, Benjamin. **Performance-based contracting for health services in developing countries: a toolkit.** Washington, DC: World Bank, 2008. 202 p. (Health, nutrition, and population series). ISBN: 978-0-8213-7536-5.
LÜCK, Heloísa. **Liderança em gestão escolar.** São Paulo: Editora Vozes, 2014. ISBN: 978-85-326-3620-1.

(UNEAL version currently is missing more disambiguate arguments, other people may want this version with author names abbreviation, others may want ISBN and others not (although one can just make this field empty in Zotero)

# Summing up

So the minimum I would want is to create a numeric version. So I ask what I would have permission to do? The options I thought:
* create a numeric version for both "Instituto Brasileiro de Informação em Ciência e Tecnologia" and UNEAL. (`instituto-brasileiro-de-informacao-em-ciencia-e-tecnologia-abnt-initials-numeric.csl`, `instituto-brasileiro-de-informacao-em-ciencia-e-tecnologia-abnt-numeric.csl` and `universidade-estadual-de-alagoas-abnt-numeric.csl`
* create `associacao-brasileira-de-normas-tecnicas-2018-numeric.csl`? but if I create this, it would be better to also have `associacao-brasileira-de-normas-tecnicas-2018.csl` (author-date). And if so, should the old version be renamed?
* other options?

**Comment by icornelius** [comment]
- re: renaming. I think the policy in this repo is to retain current names whenever possible. For an example, and the reasoning behind it, see [this commit](https://github.com/citation-style-language/styles/pull/4878/commits/f734329e45d71a56e01ac4e7ef170e5c94691af9)
- re: new styles. My experience is that the maintainers of this repo welcome new styles, provided that the styles are backed by documentation and based on an official style guide. (See [Criteria for inclusion](https://github.com/citation-style-language/styles#criteria-for-inclusion)).

**Comment by adam3smith** [comment]
Name changes are actually fine if there's a good enough reason. id/link/filename shouldn't change unless there's a very strong reason. I'll look at the other ABNT-related questions later.

**Comment by imsys** [comment]
Ok, I think just adding the year/version to the name without changing the filename would be enough.

I'm was doing a more comprehensive test on the current styles, and I found a few more inconsistencies, so I started to rewrite the csl file in a way that I think would be easier to maintain, (this maybe subject to opinion), but still reusing the work done by others.

I will work on this on a limited time this week, but as soon as I get it done I will share it.
